projects/rocsparse/clients/benchmarks/CMakeLists.txt

```
@@ -1,252 +1,82 @@
-../testings/testing_csr2hyb.cpp
-../testings/testing_csr2bsr.cpp
-../testings/testing_csr2gebsr.cpp
-../testings/testing_coo2csr.cpp
-../testings/testing_ell2csr.cpp
-../testings/testing_hyb2csr.cpp
-../testings/testing_bsr2csr.cpp
-../testings/testing_gebsr2csr.cpp
-../testings/testing_csr2csr_compress.cpp
-../testings/testing_prune_csr2csr.cpp
-../testings/testing_prune_csr2csr_by_percentage.cpp
-../testings/testing_identity.cpp
-../testings/testing_inverse_permutation.cpp
-../testings/testing_csrsort.cpp
-../testings/testing_cscsort.cpp
-../testings/testing_coosort.cpp
-../testings/testing_csricsv.cpp
-../testings/testing_csrilusv.cpp
-../testings/testing_nnz.cpp
-../testings/testing_dense2csr.cpp
-../testings/testing_dense2coo.cpp
-../testings/testing_prune_dense2csr.cpp
-../testings/testing_prune_dense2csr_by_percentage.cpp
-../testings/testing_dense2csc.cpp
-../testings/testing_csr2dense.cpp
-../testings/testing_csc2dense.cpp
-../testings/testing_coo2dense.cpp
-../testings/testing_spmv_bsr.cpp
-../testings/testing_spmv_coo.cpp
-../testings/testing_spmv_coo_aos.cpp
-../testings/testing_spmv_csr.cpp
-../testings/testing_spmv_csc.cpp
-../testings/testing_spmv_ell.cpp
-../testings/testing_v2_spmv_bsr.cpp
-../testings/testing_v2_spmv_coo.cpp
-../testings/testing_v2_spmv_coo_aos.cpp
-../testings/testing_v2_spmv_csr.cpp
-../testings/testing_v2_spmv_csc.cpp
-../testings/testing_v2_spmv_ell.cpp
-../testings/testing_v2_spmv_sell.cpp
-../testings/testing_spsv_csr.cpp
-../testings/testing_spsv_coo.cpp
-../testings/testing_sptrsv_csr.cpp
-../testings/testing_sptrsv_coo.cpp
-../testings/testing_spitsv_csr.cpp
-../testings/testing_spsm_csr.cpp
-../testings/testing_spsm_coo.cpp
-../testings/testing_sptrsm_csr.cpp
-../testings/testing_sptrsm_coo.cpp
-../testings/testing_sparse_to_dense_coo.cpp
-../testings/testing_sparse_to_dense_csr.cpp
-../testings/testing_sparse_to_dense_csc.cpp
-../testings/testing_dense_to_sparse_coo.cpp
-../testings/testing_dense_to_sparse_csr.cpp
-../testings/testing_dense_to_sparse_csc.cpp
-../testings/testing_sparse_to_sparse.cpp
-../testings/testing_extract.cpp
-../testings/testing_spgemm_bsr.cpp
-../testings/testing_spgemm_csr.cpp
-../testings/testing_spgemm_reuse_csr.cpp
-../testings/testing_spgeam_csr.cpp
-../testings/testing_spgeam_reuse_csr.cpp
-../testings/testing_spgeam_csr_2.cpp
-../testings/testing_spgeam_reuse_csr_2.cpp
-../testings/testing_gemvi.cpp
-../testings/testing_sddmm.cpp
-../testings/testing_csrcolor.cpp
-../testings/testing_check_matrix_csr.cpp
-../testings/testing_check_matrix_csc.cpp
-../testings/testing_check_matrix_coo.cpp
-../testings/testing_check_matrix_gebsr.cpp
-../testings/testing_check_matrix_gebsc.cpp
-../testings/testing_check_matrix_ell.cpp
-../testings/testing_check_matrix_hyb.cpp
-../testings/testing_check_spmat.cpp
-../testings/testing_bsrpad_value.cpp
+target_include_directories(rocsparse-bench PRIVATE
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../include>
```

```suggestion
   "${PROJECT_SOURCE_DIR}/include"
```
Would prefer to eliminate relative paths. If we move this logic somewhere else i.e. another directory it should just work.